pkgs/by-name/vu/vuetorrent/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildNpmPackage rec {
+  pname = "vuetorrent";
+  version = "2.18.0";
+
+  src = fetchFromGitHub {
+    owner = "VueTorrent";
+    repo = "VueTorrent";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-n5CtVNowGUsIHdxpnXmtXLpjwRfIoRoOr7Gh7PUE0ck=";
+  };
+
+  npmDepsHash = "sha256-PfnZ61m8VulRm18R3aqnIPrCXAy0DOOCUPIgr2eFrbk=";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/var/www
+    cp -r vuetorrent $out/var/www/vuetorrent
```

Comment:
```suggestion
    mkdir -p $out/share
    cp -r vuetorrent $out/share/vuetorrent
```

I did a `rg out/var/www/` in `nixpkgs`, and found nothing. I think `$out/share` is better.

Reply:
good point, fixing now